### ISSUE TITLE
Cannot delete namespace

### ISSUE THREAD

**Issue by melkstam** [encoredev/encore#1225]
Not always, but sometimes, the deletion of namespaces fail. It seems like switching to that namespace and then running `encore run`, going back to another namespace and then deleting it works as a workaround.

```zsh
❯ encore ns delete <namespace>
error: failed to delete namespace: could not delete volume <volume-name>: exit status 1
```

I am using Docker Desktop on macOS Sonoma.

**Comment by jakobskrym** [comment]
Heard you wanted some examples of where to improve communication @eandre , here’s an example where I would expect someone to respond 👍

**Comment by simon-johansson** [comment]
Sorry for the late reply @melkstam. Are you still encountering this issue?

**Comment by melkstam** [comment]
Yes, this is still an issue. I now use [colima](https://github.com/abiosoft/colima) but I get the same error. There are no more logs or anything in the `daemon.log` file.

**Comment by jakobskrym** [comment]
> Yes, this is still an issue. I now use [colima](https://github.com/abiosoft/colima) but I get the same error. There are no more logs or anything in the `daemon.log` file.

<img width="597" height="803" alt="Image" src="https://github.com/user-attachments/assets/e326e2a4-5a0e-4bc8-b703-21b1f62a9520" />

May be relevant 
https://discord.com/channels/814482502336905216/1455518233145638957/1455518233145638957

**Comment by ekerfelt** [comment]
I'm adding some more context to the error message:

https://github.com/encoredev/encore/pull/2328

My guess is that the container somehow doesn't exit/restarts before we try to delete the volume. Next time it happens, run `docker ps` and see if the postgres container for the affected namespace is running.
